packages/ui/test/accounts/hooks/useHasRequiredStake.test.tsx

```diff
@@ -0,0 +1,200 @@
+        locks: [],
+      },
+    }
+    const { result } = renderUseTotalBalances(1000)
+    expect(result.current).toStrictEqual({
+      hasRequiredStake: true,
+      transferableAccounts: null,
+      accountsWithLockedFounds: null,
+    })
+  })
+
+  it('Member has not enough founds', () => {
+    balance = {
+      total: new BN(10),
+      locked: new BN(0),
+      transferable: new BN(10),
+      recoverable: new BN(0),
+      locks: [],
+    }
+    balances = {
+      [alice.address]: balance,
+      [aliceStash.address]: {
+        total: new BN(50),
+        locked: new BN(0),
+        transferable: new BN(50),
+        recoverable: new BN(0),
+        locks: [],
+      },
+      [bob.address]: {
+        total: new BN(20),
+        locked: new BN(20),
+        transferable: new BN(0),
+        recoverable: new BN(0),
+        locks: [],
+      },
+    }
+    const { result } = renderUseTotalBalances(1000)
+    expect(result.current).toStrictEqual({
+      hasRequiredStake: false,
+      transferableAccounts: null,
+      accountsWithLockedFounds: null,
+    })
+  })
+
+  it('Not enough funds with transferrable funds', () => {
+    balance = {
+      total: new BN(0),
+      locked: new BN(0),
+      transferable: new BN(0),
+      recoverable: new BN(0),
+      locks: [],
+    }
+    balances = {
+      [alice.address]: balance,
+      [aliceStash.address]: {
+        total: new BN(900),
+        locked: new BN(0),
+        transferable: new BN(900),
+        recoverable: new BN(0),
+        locks: [],
+      },
+      [bob.address]: {
+        total: new BN(100),
+        locked: new BN(0),
+        transferable: new BN(100),
+        recoverable: new BN(0),
+        locks: [],
+      },
+    }
+    const { result } = renderUseTotalBalances(1000)
+    expect(result.current).toStrictEqual({
+      hasRequiredStake: false,
+      transferableAccounts: [aliceStash, bob],
+      accountsWithLockedFounds: null,
+    })
+  })
+
+  it('Not enough funds with transferrable to an account with locked founds', () => {
+    balance = {
+      total: new BN(50),
+      locked: new BN(0),
+      transferable: new BN(50),
+      recoverable: new BN(0),
+      locks: [],
+    }
+    balances = {
+      [alice.address]: balance,
+      [aliceStash.address]: {
+        total: new BN(250),
+        locked: new BN(200),
+        transferable: new BN(50),
+        recoverable: new BN(0),
+        locks: [],
+      },
+      [bob.address]: {
+        total: new BN(700),
+        locked: new BN(400),
+        transferable: new BN(300),
+        recoverable: new BN(0),
+        locks: [],
+      },
+    }
+    const { result } = renderUseTotalBalances(1000)
+    expect(result.current).toStrictEqual({
+      hasRequiredStake: false,
+      transferableAccounts: null,
+      accountsWithLockedFounds: {
+        [bob.address]: [aliceStash],
+        [aliceStash.address]: [bob],
+        [alice.address]: [aliceStash, bob],
+      },
+    })
```

Comment:
```typescript
    balances = {
      [alice.address]: {
      	...zeroBalance,
      	total: new BN(50),
      	transferable: new BN(50),
    	},
      [aliceStash.address]: {
        ..zeroBalance,
    	  total: new BN(250),
        locked: new BN(200),
        transferable: new BN(50),
      },
      [bob.address]: {
        total: new BN(700),
        locked: new BN(400),
        transferable: new BN(300),
      },
    }
    const { result } = renderUseTotalBalances(1000)
    expect(result.current).toStrictEqual({
      hasRequiredStake: false,
      transferableAccounts: null,
      accountsWithLockedFounds: {
        [bob.address]: [aliceStash],        // 700 + (50) < 1000
        [aliceStash.address]: [bob],        // 250 + (300) < 1000
        [alice.address]: [aliceStash, bob], // 50 + (50 + 300) < 1000
      },
    })
```

The numbers in the test setup are correct :+1:. However, the transfer logic is different -> you should only take the `transferrable` amount to check if, when transferred to another account, will give an expected total (`my total + other transferable`).

I think that I might provide you some input forgetting to explain that the "locked" amount can't be transferred.

Also, I've found [a bug](https://github.com/Joystream/pioneer/issues/720) in our calculation of total / transferable amount :(